docs/reference/modules/node.asciidoc

```diff
@@ -32,42 +33,52 @@ configure this setting, the node has the following roles by default:
 * `remote_cluster_client`
 * `transform`
 
-NOTE: If you set `node.roles`, the node is assigned only the roles you specify.
+[IMPORTANT]
+====
+If you set `node.roles`, ensure you specify every node role your cluster needs.
+To access most {stack} features, a minimal cluster requires nodes with the
+following roles:
+
+- `master`
+- `ingest`
+- `transform`
+- `data`, `data_content`, or a combination of the `data_hot`, `data_warm`,
```

Comment:
`data_content` should be part of second clause in the disjunction. A node either has the `data` role or it has specific `data_` roles. `data_content` is another tier in the data tiering.